src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const content = document.querySelectorAll('.population');
+let total = 0;
+let avarage = 0;
+const arr = [];
```

Comment:
It's good to make var names more representative, so that it's not needed to look up what you put in this var later to figure out what is it supposed to hold
```suggestion
const populationArray = [];
```